src/main/starlark/core/repositories/BUILD

```diff
@@ -39,7 +39,7 @@ bzl_library(
     name = "repositories",
     srcs = glob(["*.bzl"]),
     visibility = ["//:__subpackages__"],
-    deps = [
-        "@rules_proto//proto:defs",
-    ],
+    #    deps = [
```

Comment:
Did you mean to leave this commented out?

Reply:
Unused dep.